testrunner/execute.go

```diff
@@ -230,6 +232,16 @@ func removeObsoleteParentTests(tests []testResult) []testResult {
 	return results
 }
 
+func formatTestNames(tests []testResult) []testResult {
+	var out []testResult
+	for _, test := range tests {
+		re := regexp.MustCompile("[/_]")
```

Comment:
Regexes only need to be compiled once. Here the regex is being compiled in each loop iteration. A better alternative would be to compile the regex before the loop and use the compiled result as many times as needed inside the loop. This way the regex is only compiled once per function call. It is also common to take this a step further and compile regex in init blocks, which makes the regex only be compiled once during the lifetime of the program.

However, I don't think regexes are really needed here (see comment below).